miso-web/src/main/webapp/pages/editPool.jsp

```diff
@@ -229,56 +229,8 @@
 <!-- notes end -->
 
 <c:if test="${pool.id != 0}">
-<miso:qcs id="list_qcs" item="${pool}"/>
-</c:if>
-
-<div class="sectionDivider" onclick="Utils.ui.toggleLeftInfo(jQuery('#experiments_arrowclick'), 'experiments');">
-  Experiments
-  <div id="experiments_arrowclick" class="toggleLeftDown"></div>
-</div>
-<div id="experiments" style="display:none;">
-<h1>Experiments</h1>
-
-<div class="note">
-  <h2>Selected experiment(s):</h2>
-
-  <div id="exptlist" class="elementList ui-corner-all">
-    <c:if test="${not empty pool.experiments}">
-      <c:forEach items="${pool.experiments}" var="exp">
-        <div onMouseOver="this.className='dashboardhighlight'" onMouseOut="this.className='dashboard'"
-             class="dashboard">
-          <span class='float-left'>
-          <input type="hidden" id="experiments${exp.id}" value="${exp.id}" name="experiments"/>
-          <b>Experiment:</b> <a href="<c:url value="/miso/experiment/${exp.id}"/>">${exp.alias} (${exp.name})</a><br/>
-          <b>Description:</b> ${exp.description}<br/>
-          <b>Project:</b> <a
-              href="<c:url value="/miso/project/${exp.study.project.id}"/>">${exp.study.project.alias}
-            (${exp.study.project.name})</a><br/>
-          </span>
-          <span onclick='Utils.ui.confirmRemove(jQuery(this).parent());'
-                class='float-right ui-icon ui-icon-circle-close'></span>
-        </div>
-      </c:forEach>
-    </c:if>
-    <input type="hidden" value="on" name="_experiments"/>
-  </div>
-</div>
-<h2 class="hrule">Select experiments:</h2>
-<table class="in">
-  <tr>
-    <td style="vertical-align:top;width:30%">
-      <label for="selectExpts"><b>Search experiments:</b></label><br/>
-      <input type="text" id='selectExpts' name="selectExpts" value=""
-             onKeyup="Utils.timer.timedFunc(Pool.search.poolSearchExperiments(this, 'ILLUMINA'),200);"/>
-
-      <div id='exptresult'></div>
-    </td>
-  </tr>
-</table>
-</div>
-<br/>
-
-<c:if test="${pool.id != 0}">
+  <miso:qcs id="list_qcs" item="${pool}"/>
+  <miso:list-section id="list_experiment" name="Experiments" target="experiment" alwaysShow="true" items="${experiments}" config="{ poolId: ${pool.id} }"/>
```

Comment:
I'm not sure why, but this isn't showing up regardless of whether there is an experiment for the pool

Reply:
Probably because `${experiments}` is not defined and this shouldn't be on the pool page any more. I've moved it to Edit Library.